braindecode/models/base.py

```diff
@@ -0,0 +1,112 @@
+from typing import List
+
+
+class EEGModuleMixin:
+    """
+    Mixin class for EEG models.
+
+    Parameters
+    ----------
+    n_channels: int
+        Number of EEG channels.
+    channel_names: list of str
```

Comment:
I would use ch_names to be consistent with mne and some other places in braindecode